server/src/main/java/org/elasticsearch/cluster/metadata/LifecycleExecutionState.java

```diff
@@ -57,7 +57,7 @@ public record LifecycleExecutionState(
     private static final String SNAPSHOT_REPOSITORY = "snapshot_repository";
     private static final String SNAPSHOT_INDEX_NAME = "snapshot_index_name";
     private static final String SHRINK_INDEX_NAME = "shrink_index_name";
-    private static final String ROLLUP_INDEX_NAME = "rollup_index_name";
+    private static final String DOWNSAMPLE_INDEX_NAME = "rollup_index_name";
```

Comment:
Kept old value to avoid making this a breaking change.
The 8.7 branch has been created and we should avoid breaking changes. 
The main reason was to rename classes and field/parameter/variable names.